ClientManagementService/ClientManagementService.API/DTOMapper/PetDTOMapper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using ClientManagementService.API.DTO;
+using ClientManagementService.Domain.Models;
+using CorePet = ClientManagementService.Domain.Models.Pet;
+
+
+namespace ClientManagementService.API.DTOMapper
+{
+    public class PetDTOMapper
+    {
+        public static PetDTO ToDTOPet(CorePet corePet)
+        {
+            var dtoPet = new PetDTO();
+
+            dtoPet.Id = corePet.Id;
+            dtoPet.OwnerId = corePet.OwnerId;
+            dtoPet.BreedId = corePet.BreedId;
+            dtoPet.Name = corePet.Name;
+            dtoPet.Dob = corePet.Dob;
+            dtoPet.Weight = corePet.Weight;
+            dtoPet.BordetellaVax = corePet.BordetellaVax;
+            dtoPet.Dhppvax = corePet.Dhppvax;
+            dtoPet.RabieVax = corePet.RabieVax;
+            dtoPet.OtherInfo = corePet.OtherInfo;
+            dtoPet.Picture = corePet.Picture;
+            dtoPet.OwnerFirstName = corePet.Owner.FirstName;

```

Comment:
This could be problematic. Reason is because corePet.Owner will be null.

Same thing with corePet.BreedInfo.

When you do gets, you didn't do the "Include". However, "Include" can be quite slow.